src/ResourceManager/CognitiveServices/Commands.Management.CognitiveServices/CognitiveServicesAccount/CognitiveServicesAccountBaseCmdlet.cs

```diff
@@ -32,6 +32,7 @@ public abstract class CognitiveServicesAccountBaseCmdlet : AzureRMCmdlet
         protected const string CognitiveServicesAccountNounStr = "AzureRmCognitiveServicesAccount";
```

Comment:
This has to be changed to:
```suggestion
         protected const string CognitiveServicesAccountNounStr = ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "CognitiveServicesAccount";
```
If not, this won't work properly for `Az`.

Reply:
coll, also this `suggest change` feature is cool.